ovirt_metrics.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
 
   # HACK: Rails 5 dropped support for Ruby < 2.2.2
   active_gems_version = "< 5" if Gem::Version.new(RUBY_VERSION) <= Gem::Version.new("2.2.2")
-  spec.add_dependency "activerecord", ">= 4.2.3", (active_gems_version || "< 5.2")
```

Comment:
Can you remove this entire hack since we're dropping Rails 4.2 support?  Unfortunately it didn't actually work the way we intended it to.

Reply:
Sure! Shame.